src/app/resources/resources-add.component.html

```diff
@@ -75,7 +75,7 @@
           </mat-select>
         </mat-form-field>
         <mat-form-field>
-          <input matInput [(ngModel)]="currentDate" formControlName="articleDate" i18n-placeholder placeholder="Article Date" readonly>
+          <input matInput value="{{resourceForm.value.articleDate | date}}" i18n-placeholder placeholder="Article Date" readonly>
```

Comment:
We have used date: 'fullDate' on most of the component. So, could we do same on everywhere ?